Nos/Views/NoteButton.swift

```diff
@@ -13,8 +13,8 @@ struct NoteButton: View {
     var style = CardStyle.compact
     var shouldTruncate: Bool
     var hideOutOfNetwork: Bool
-    var showReplyCount: Bool
-    var displayRootMessage: Bool 
+    var replyCount: ReplyCount?
```

Comment:
Also I would find it more expressive to have `.none` be a style instead of using `nil`.

Reply:
I made a suggestion around the same time you did -- I think we're aligned on direction and just need a good name. I suggested `repliesDisplayType` since I prefer not to have `count` in the name. I'm open to other options, though.

Reply:
Good suggestions! I renamed it to `RepliesDisplayType` and added a `.none` case